test/metabase/api/table_test.clj

```diff
@@ -1038,6 +1028,32 @@
                table    (upload-test/create-upload-table!)]
            (is (.exists file) "File should exist before replace-csv!")
            (mt/with-current-user (mt/user->id :crowberto)
-             (replace-csv! {:table-id (:id table)
-                            :file     file}))
+             (update-csv! {:action   :metabase.upload/replace
+                           :table-id (:id table)
+                           :file     file}))
            (is (not (.exists file)) "File should be deleted after replace-csv!")))))))
+
+;;; +----------------------------------------------------------------------------------------------------------------+
+;;; |                                          DELETE /api/table/:id
+;;; +----------------------------------------------------------------------------------------------------------------+
+
+(defn- delete-csv! [table-id]
+  (@#'api.table/delete-csv! table-id))
+
+(defn- delete-csv-via-api! [table-id]
```

Comment:
hmm, this is a bit misleading IMO, this is not doing things via API, but rather function in API ns. 

Also, I'm not sure I see the benefit of doing this versus calling `mt/user-http-request` directly.

Reply:
Naming comes from consistency with `create-` and `update-` routes. How about I change those to `-via-api-handler!`?

Reply:
Done